vips/vip-501/bscmainnet.ts

```diff
@@ -0,0 +1,346 @@
+  };
+  interestRateModel: {
+    address: string;
+    base: string;
+    multiplier: string;
+    jump: string;
+    kink: string;
+  };
+};
+
+export const weETHMarket: Market = {
+  vToken: {
+    address: "0x0170398083eb0D0387709523baFCA6426146C218",
+    name: "Venus weETH (Core)",
+    symbol: "vweETH_Core",
+    underlying: weETH,
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.7", 18),
+    liquidationThreshold: parseUnits("0.75", 18),
+    supplyCap: parseUnits("4000", 18),
+    borrowCap: parseUnits("400", 18),
+    reserveFactor: parseUnits("0.4", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("3.614915477041407445", 18),
+    vTokensToBurn: parseUnits("0.03694", 8), // around $100
+    vTokenReceiver: unichainmainnet.VTREASURY,
+  },
+  interestRateModel: {
+    address: "0xc2edB1e96e45178b901618B96142C2743Ed0e7B3",
+    base: "0",
+    multiplier: "0.09",
+    jump: "3",
+    kink: "0.45",
+  },
+};
+
+export const wstETHMarket: Market = {
+  vToken: {
+    address: "0xbEC19Bef402C697a7be315d3e59E5F65b89Fa1BB",
+    name: "Venus wstETH (Core)",
+    symbol: "vwstETH_Core",
+    underlying: wstETH,
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.7", 18),
+    liquidationThreshold: parseUnits("0.725", 18),
+    supplyCap: parseUnits("14000", 18),
+    borrowCap: parseUnits("7000", 18),
+    reserveFactor: parseUnits("0.25", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("3.208916246034338443", 18),
+    vTokensToBurn: parseUnits("0.03309", 8), // around $100
+    vTokenReceiver: unichainmainnet.VTREASURY,
+  },
+  interestRateModel: {
+    address: "0x5C7D8858a25778d992eE803Ce79F1eff60c1d9D1",
+    base: "0",
+    multiplier: "0.15",
+    jump: "3",
+    kink: "0.45",
+  },
+};
+
+const vip501 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-501 [Unichain] Add weETH and wstETH markets to the Core pool",
+    description: "",
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      // <--- weETH Market --->
+      // Oracle config
+      {
+        target: unichainmainnet.REDSTONE_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[weETH.address, WEETH_REDSTONE_FEED, STALE_PERIOD_26H]],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: unichainmainnet.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3],bool))",
```

Comment:
We could use `setTokenConfigs` instead, to configure the oracle for wstETH and weETH in one command

Reply:
[aab67ad](https://github.com/VenusProtocol/vips/pull/561/commits/aab67ad2943b156c60e262af3104d35cf9a9933e)